dio/lib/diox.dart

```diff
@@ -17,3 +17,4 @@ export 'src/parameter.dart';
 export 'src/redirect_record.dart';
 export 'src/response.dart';
 export 'src/transformer.dart';
+export 'src/transformers/background_transformer/background_transformer.dart';
```

Comment:
Any specific reason that we need to have `/background_transformer` directory here?

Reply:
No, previously the compute function was also in this folder. When moving it, I must have forgotten about this folder.